tensorboard/uploader/uploader_subcommand.py

```diff
@@ -438,7 +438,7 @@ def execute(self, server_info, channel):
         print(
             "Upload started and will continue reading any new data as it's added"
         )
-        print("to the logdir. To stop uploading, press Ctrl-C.")
+        print("to the logdir.\n\nTo stop uploading, press Ctrl-C.")
```

Comment:
It strikes me as a bit weird to structure the print statements this way, where we have some explicit newlines in the text now, in addition to the one that's implicitly inserted before "to the logdir" by virtue of using a separate `print()` call.  As the PR description shows, that leads to sort of stranding "to the logdir" on its own line with a hard line-wrap, which is inconsistent with the hparams message above it which isn't line-wrapped:

```
Data for the "hparams" plugin is now uploaded to TensorBoard.dev! Note that uploaded data is public. If you do not want to upload data for this plugin, use the "--plugins" command line argument.
Upload started and will continue reading any new data as it's added
to the logdir.

To stop uploading, press Ctrl-C.
```

It would seem better to just avoid line-wrapping that part:

```
Data for the "hparams" plugin is now uploaded to TensorBoard.dev! Note that uploaded data is public. If you do not want to upload data for this plugin, use the "--plugins" command line argument.
Upload started and will continue reading any new data as it's added to the logdir.

To stop uploading, press Ctrl-C.
```

Could be written in code as

```
print(
    "Upload started and will continue reading any new data as it's added"
    "to the logdir."
)
print()
print("To stop uploading, press Ctrl-C.")
```

or

```
print(
    "Upload started and will continue reading any new data as it's added"
    "to the logdir.\n\nTo stop uploading, press Ctrl-C."
)
```